package/MDAnalysis/analysis/pcz.py

```diff
@@ -0,0 +1,481 @@
+        associated with the supplied universe, subject to the (optional) 
+        atom and frame selections. The selected atoms are least-squares 
+        fitted using an iterative procedure to remove global rotation and 
+        translation before the covariance matrix is calculated. 
+        If reference_coordinates, in the form of a [N, 3] numpy array (where 
+        N is the number of selected atoms) is provided, this will be used to 
+        initialise the least-squares fitting process, otherwise the first 
+        snapshot in the trajectory will be used.
+
+        :Arguments:
+          *universe*
+            :class:`MDAnalysis.Universe`
+
+          *atom_selection* 
+            a valid selection string for 
+            :meth:`MDAnalysis.core.AtomGroup.AtomGroup.select_atoms`
+
+          *start*
+            int, the index (zero-based) of the first frame in the trajectory
+            to include in the PC analysis
+
+          *stop*
+            int, the index (zero-based) of the last frame in the trajectory 
+            to include in the PC analysis
+
+          *step*
+            int, the increment between frames to be included in the analysis
+
+          *fitting_tolerance*
+            float, the iterative least-squares fitting process will terminate
+            when the average structure has an RMSD less than this from the
+            structure generated in the previous cycle (or if *max_iterations*
+            is reached first).
+
+          *max_iterations*
+            int, the maximum number of least-squares fitting cycles
+
+          *captured_variance*
+            int or float, if < 1 then enough eigenvectors will be kept to
+            explain *captured_variance* fraction of the total variance, if
+            >=1 (and integral), this is the number of eigenvectors that will
+            be kept. Providing this argument permits the precalculation of 
+            projections, which can improve performance.
+        """
+
+        # Step 1: Create the "Fasu" object from the selected trajectory data.
+        self._f = Fasu(universe, atom_selection=atom_selection, start=start, stop=stop, step=step)
+        self.natoms = self._f.natoms
+        self.numframes = self._f.numframes
+
+        # Step 2: Calculate the mean structure, later used to remove global
+        #         translation and rotation from the trajectory frames before
+        #         the covariance matrix calculation. This iterative method
+        #         is based on the "Procrustes" approach (Dryden and Mardia,
+        #         "Statistical Shape Analysis", ISBN 978-0-471-95816-1).
+
+        if reference_coordinates is None:
+            reference_coordinates = self._f.coords(0)
+
+        rms = fitting_tolerance + 1.0
+        old_avg = reference_coordinates
+        new_avg = reference_coordinates
+        iteration = 0
+        while rms > fitting_tolerance and iteration < max_iterations:
```

Comment:
The least squares fitting is necessary to remove global rotation and translation before the covariance matrix is calculated, otherwise the PCA components will take into account the rotation/translation motion of the molecules that might be the dominant trend, obscuring therefore the interesting part of the molecular parts motions. We would generally be interested on only the motion of the parts of the molecule and that is why we perform least-squares fitting in advance. This method has been validated/used in different papers that use PCA such as:
[Wlodek1997] Wlodek ST, Clark TW, Scott LR, McCammon JA (1997) J Am Chem Soc 119, 9513-9522.
[Sherer1999] Sherer EC, Harris SA, Soliva R, Orozco M, Laughton CA (1999) J Am Chem Soc 121, 5981-5991.